scripts/e2e/test-prohibit-backend.yaml

```diff
@@ -0,0 +1,61 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: brownfield-blacklist-deployment
+  namespace: test-brownfield-ns
+spec:
+  selector:
+    matchLabels:
+      app: ws-app
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: ws-app
+    spec:
+      containers:
+        - name: brownfield-blacklist-app
+          imagePullPolicy: Always
+          image: docker.io/kennethreitz/httpbin
+          ports:
+            - containerPort: 80
+          livenessProbe:
+            httpGet:
+              path: /status/200
+              port: 80
+            initialDelaySeconds: 3
+            periodSeconds: 3
+      imagePullSecrets:
+        - name: acr-creds
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: brownfield-blacklist-service
+  namespace: test-brownfield-ns
+spec:
+  selector:
+    app: ws-app
+  ports:
+  - protocol: TCP
+    port: 80
+    targetPort: 80
+---
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: brownfield-blacklist-ingress
+  namespace: test-brownfield-ns
+  annotations:
+    kubernetes.io/ingress.class: azure/application-gateway
+    appgw.ingress.kubernetes.io/backend-path-prefix: "/"
+spec:
+  rules:
+    - host: brownfield-blacklist-ns.host
```

Comment:
host here doesn't match the hostname in the above .yaml

brownfield-blacklist.host != brownfield-blacklist-ns.host